translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,421 @@
+
```

Comment:
Several of these methods seem to be the same as in `icone_translator.py`. Let's create a shared library that we can pull in to both of those scripts and reuse the code.

Reply:
i created a shared library in source_code directory.